apps/test/unit/storage/dataUtilsTest.js

```diff
@@ -227,7 +227,8 @@ describe('castValue', () => {
     it('only allows unquoted strings if allowUnquotedStrings is true', () => {
       expect(castValue('abc', /* allowUnquotedStrings */ true)).to.equal('abc');
       expect(() => castValue('abc', /* allowUnquotedStrings */ false)).to.throw(
-        /JSON Parse error/
+        //      PhantomJS|Chrome
+        /JSON Parse error|Unexpected token/
```

Comment:
Chrome and Phantom simply return different exception text in their JSON parse error.